policyengine_us/variables/gov/states/ny/nysdps/heat_pump_category.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+class HPCategory(Enum):
+    C2 = "ccASHP: Full Load Heating"
+    C2A = "ccASHP: Full Load Heating with integrated controls (inclusive of base incentive)"
+    C2B = "ccASHP: Full Load Heating with decommissioning (inclusive of base incentive)"
+    C2E = "Air-to-Water Heat Pump for space conditioning"
+    C3 = "GSHP: Full Load Heating"
+    C4 = "Custom Space Heating Applications"
+    C4A1 = "Heat Pump + Envelope Tier 1"
+    C4A2 = "Heat Pump + Envelope Tier 2"
+    C5A = "Residential Rated HPWH: Retail" 
+    C5B = "Residential Rated HPWH: Midstream"
+    C6 = "Custom Hot Water Heating Applications"
+    C7 = "GSHP Desuperheater"
+    C8 = "WWHP used for DHW"
+    C9 = "Simultaneous Installation of Space Heating & Domestic Water Heating"
+
+
+class heat_pump_category(Variable):
```

Comment:
```suggestion
class ny_clean_heat_heat_pump_category(Variable):
```

Reply:
all program related variables  - same format

+ please add references to each of the variables to show where the definitions originated